TM1py/Services/ServerService.py

```diff
@@ -121,10 +121,11 @@ def get_message_log_entries(self, reverse: bool = True, since: datetime = None,
 
             if msg_contains:
                 if isinstance(msg_contains, str):
-                    log_filters.append(format_url("contains(Message,'{}')", msg_contains))
+                    log_filters.append(format_url("contains(toupper(Message),toupper('{}'))", msg_contains))
                 else:
-                    msg_filters = [format_url("contains(Message,'{}')", wildcard) for wildcard in msg_contains]
-                    log_filters.append("({})".format(" and ".join(msg_filters)))
+                    msg_filters = [format_url("contains(toupper(Message),toupper('{}'))", wildcard) 
+                                   for wildcard in msg_contains]
+                    log_filters.append("({})".format(" or ".join(msg_filters)))
```

Comment:
Now we are not in line with the docstring of the function (`list of substrings will be queried as AND statement`) and the change may break compatibility.

Perhaps we can add an optional argument (something like `operator='AND'`) to the function to control if AND or OR should be used. What do you think?

Reply:
Yeah that should be useful!